src/logger/settings.ts

```diff
@@ -10,14 +10,20 @@ export const SOURCE_TRACKING_CONFIG_PROP = "MTA Tools.sourceLocationTracking";
  * @returns {LogLevel}
  */
 export function getLoggingLevelSetting(): string {
-  return vscode.workspace.getConfiguration().get(LOGGING_LEVEL_CONFIG_PROP);
+  return (
+    vscode.workspace.getConfiguration().get(LOGGING_LEVEL_CONFIG_PROP) ??
```

Comment:
Optional - we might want to align the configuration with the `package.json`. For example, change it to `error` here: https://github.com/SAP/vscode-mta-tools/blob/7173a1a6385d2c1148b95e85198eeff682542f5e/package.json#L101

Reply:
Done